src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -693,7 +703,7 @@ private List<String> doRegularElimination(
     LinkedList<String> lastPlaceCandidates = currentRoundTallyToCandidates.get(minVotes);
     if (lastPlaceCandidates.size() > 1) {
       // there was a tie for last place
-      // create new Tiebreak object to pick a loser
+      // create new Tiebreak object to pick a winner
```

Comment:
Not clear why you changed this comment. This Tiebreak object is selecting a loser, not a winner.